nevergrad/benchmark/xpbase.py

```diff
@@ -57,15 +59,20 @@ def __init__(
         self.optimizer = optimizer
         self.budget = budget
         self.num_workers = num_workers
-        self.executor = execution.MockedTimedExecutor(batch_mode)
+        ng_cpus_per_task = int(os.getenv("NG_CPUS_PER_TASK", "1"))
+        if ng_cpus_per_task > 1:
+            self.executor: tp.Any = futures.ThreadPoolExecutor(max_workers=ng_cpus_per_task)
```

Comment:
I'm still very much against this since it
- prevents reproducibility
- adds complexity which may create a lot of issues (eg: not sure that it will checkpoint correctly if preempted)
- assumes the benchmarks are threadsafe, which is not always true on our side and possibly on the dependency side

Can you quantify the speed gain?
Python executes one thread at a time for most operations, so it may just waste time for creating threads without going faster (depending on what is happening in the underlying code)
Moving to a ProcessPoolExecutor could help for parallelization, but I would expect a big overhead so that it could be unhelpful most of the time